development/workflow.rst

```diff
@@ -0,0 +1,305 @@
+#######################################
+DM's Collaborative Development Workflow
+#######################################
+
+This document describes our procedures for collaborating on LSST DM software and documentation development with `Git <http://git-scm.org>`_, `GitHub <https://github.com>`_ and `JIRA <https://jira.lsstcorp.org/>`_.
+
+New LSST developers can also read the :doc:`lsstsw Stack Development Tutorial </development/lsstsw_tutorial>` to see how these processes work in a holistic stack development workflow.
+
+In appendices we also give suggestions :ref:`setting your Git environment <git-setup-best-practices>`, and general best practices for :ref:`organizing git commits <git-commit-organization-best-practices>` and :ref:`writing great commit messages <git-commit-message-best-practices>`.
+
+.. _git-setup:
+
+Git & GitHub Setup
+==================
+
+DM developers need to install Git version 1.8.2 or later and the Git LFS client to work with our LFS-backed data repositories.
+See the :doc:`DM Git LFS documentation </development/git_lfs>` for more information on how to install and setup the Git LFS client for your machine.
+
+Ensure that Git is setup to use your *institution-hosted* email address the :file:`~/.gitconfig` file.
+You can do this from the command line:
+
+.. code-block:: bash
+   git config --global user.name "Your Name"
+   git config --global user.email "your_email@institution.edu"
+
+Likewise, in your `GitHub account email settings <https://github.com/settings/emails>`_, add your institution-hosted email (e.g. @lsst.org).
+Set this institutional email as your **Primary GitHub** email address.
```

Comment:
Why does it have to be primary? I might not want it to be my primary github address. Github is perfectly capable of associating addresses with different organizations so my personal email address is my primary address but all LSST org traffic goes to my LSST email account.